__tests__/integration/previewSaveResource.test.js

```diff
@@ -121,7 +122,5 @@ describe('Preview and save resource', () => {
       return btn.id !== 'modal-save' && btn.id !== 'editor-save'
     })
     fireEvent.click(finalSave)
```

Comment:
Test that modal closed?

Reply:
Done, added test for confirming modal in state is undefined.